gnomad/utils/annotations.py

```diff
@@ -1140,65 +1168,68 @@ def hemi_expr(
     )
 
 
-def variant_report(
-    table_to_parse: hl.Table,
+def get_variant_json(
+    ht: hl.Table,
     variant: str,
     output_path: str,
     template: str = "",
     ref_input: str = "GRCh38",
 ) -> None:
     """
-    Thise code reads in
-    1) A reference table to parse containing the variant of interest and its VRS Annotations
-    2) A string of the Chr, Pos, Ref, and Alt of a Variant
-    3) A GA4GH-VRS JSON Template
-
-    and reports back a JSON containing the GA4GH-VRS Information of the variant
-
-    prints it and reports it to the desired location
+    Filter to a specified variant and return a JSON string containing the GA4GH-VRS annotations.
 
     """
-    import json
 
     chr_in, pos_in, ref_in, alt_in = variant.split("-")
-    table_filtered = table_to_parse.filter(
-        table_to_parse.locus
-        == hl.locus(contig=chr_in, pos=int(pos_in), reference_genome=ref_input)
+    # build_in = gnomad.utils.filtering.get_reference_genome(ht.locus).name
```

Comment:
add 'from gnomad.utils.filtering import get_reference_genome' to imports at top of script